symengine/functions.h

```diff
@@ -1037,6 +1037,55 @@ class Abs: public Function {
 
 //! Canonicalize Abs:
 RCP<const Basic> abs(const RCP<const Basic> &arg);
+
+class Max: public Function {
+
+private:
+    vec_basic arg_;
+public:
+    IMPLEMENT_TYPEID(MAX)
+    //! Max Constructor
+    Max(const vec_basic&& arg);
+    Max(const RCP<const Basic> &arg, ...);
```

Comment:
Is this implemented? With the first constructor, you can do `Max({x, y, z})` right?